### ISSUE TITLE
TTv5 Q&A: Color Contrast Issue

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#198]
Location: Sign-up for Trusted Tester Q&A - Tuesday Webinar page
URL: https://dev.section508testing.net/mod/facetoface/signup.php?s=32&backtoallsessions=15
Issue: Question mark (?) for help with notification type has a color contrast ratio (2.8:1) that is less than 4.5:1.  This is global issue, so it should be fixed wherever this text appears within the application.

![image](https://user-images.githubusercontent.com/34749031/60452559-ca52d200-9bfc-11e9-9b11-c8efbb23413a.png)

**Comment by drewnielson** [comment]
fixed - added filter to .iconhelp class in clean css file
